docs/usage.md

```diff
@@ -47,6 +47,7 @@ If you are using a config file, replace any dashes (`-`) with an underscore (`_`
 | `--quiet or -q {ok,ignore,warning,error}`             | Suppress certain output messages, instead of writing them to stdout.                                                                                                                                            |
 | `--warnings-as-errors [MIGRATION_TEST_CODE [...]]`    | Handle warnings as errors and therefore return an error status code if we should. Optionally specify migration test codes to handle as errors. When no test code specified, all warnings are handled as errors. |
 | `--sql-analyser`                                      | Specify the SQL analyser that should be used. Allowed values: 'sqlite', 'mysql', 'postgresql'.                                                                                                                  |
+| `--analyser-string-mapping`                           | Specify a JSON dict of custom SQL analysers with their fully qualified class names. Provide --sql-analyser key to use a custom analyser.                                                                        |

```

Comment:
We should use American english, rather than British, for this flag.

```suggestion
| `--analyser-string-mapping`                           | Specify a JSON dict of custom SQL analysers with their fully qualified class names. Provide --sql-analyzer key to use a custom analyzer.                                                                        |
```